Sources/SwiftTemplate.swift

```diff
@@ -3,9 +3,16 @@
 // Copyright (c) 2017 SwiftGen
 // MIT Licence
 //
-
+import Foundation
 import Stencil
 
+#if os(Linux)
+#if swift(>=3.1)
```

Comment:
Is it a branch that is based on @krzysztofzablocki 's `feature/spm-support` branch of this PR or a separate branch from `develop`? (Sorry I'm in the train, not enough connection for git fetch to work reliably to check myself 😄)

@djbe This branch is on the repo (and not a fork), and you both have ( @krzysztofzablocki + you) push access on this repo, so you should be able to add commits to this branch instead of creating a separate one, any reason not to here?